docs/computing/containers/tykky.md

```diff
@@ -20,79 +27,80 @@ are not covered here yet.
 !!! Warning
     As Tykky is still under development some of the more advanced features might change in exact usage and API.
 
-## Basic conda installation
+## Tykky module
 
-To access the tools provided by Tykky, simply load the module, `module load tykky`
+To access Tykky tools: 
 
+1) Usually it is best to first unload all other modules: 
 
-Then we can run:
-```bash
-conda-containerize new --prefix <install_dir> env.yml
+```
+module purge
 ```
 
-Where **env.yml** is a conda environment file.
-This file can be written manually, e.g:
+2) Load Tykky module. 
 
-env.yaml
-```yaml
-channels:
-  - conda-forge
-dependencies:
-  - python=3.8.8
-  - scipy
-  - nglview
+```bash
+module load tykky
 ```
 
-Or generate them from an existing environment
+## Conda based installation
 
-```
-conda env export -n <target_env_name> > env.yaml 
-```
-_Windows and MacOS will need to add the `--from-history` flag to the export command_
+First make sure that you have read and understood the license terms for miniconda and any used channels
+before using the command. 
 
-or 
-```
-conda list -n <target_env_name> --explicit > env.txt
+- [Miniconda end user license agreement](https://www.anaconda.com/end-user-license-agreement-miniconda)
+- [Anaconda terms of service](https://www.anaconda.com/terms-of-service)
+- [A blog entry on Anaconda commercial edition](https://www.anaconda.com/blog/anaconda-commercial-edition-faq)
+
+1) Create **conda environment file** env.yml: 
```

Comment:
An example `env.yaml` may be useful for the user here.

Reply:
Also `conda env export -n <target_env_name> > env.yaml` quick instructions to export an environment.